examples/mobilenet/MobileNet.js

```diff
@@ -53,6 +57,22 @@ class MobileNet {
     return 'success';
   }
 
+  _getPrefer() {
+    let prefer = this._nn.PREFER_FAST_SINGLE_ANSWER;
```

Comment:
Please use PREFER_SUSTAINED_SPEED (MPS) by default

Reply:
Yes, for MacOS platform the default preference is set as PREFER_SUSTAINED_SPEED (MPS). Here I set  PREFER_FAST_SINGLE_ANSWER default for others platform.